activerecord/lib/active_record/associations.rb

```diff
@@ -1244,7 +1244,7 @@ module ClassMethods
         #   Serves as a composite foreign key. Defines the list of columns to be used to query the associated object.
         #   This is an optional option. By default Rails will attempt to derive the value automatically.
         #   When the value is set the Array size must match associated model's primary key or +query_constraints+ size.
-        # [:index_errors]
+        # [+:index_errors]
```

Comment:
```suggestion
        # [+:index_errors+]
```

Checkout the `docs-preview` job in CI, you can see what the change will look like when published:

![image](https://github.com/user-attachments/assets/5514a83e-36c3-4eec-9fa6-a3315f7a574a)

Reply:
You are right, didn't pay enough attention to that.